src/css/support.js

```diff
@@ -96,7 +96,12 @@ define([
 			div.removeChild( marginDiv );
 
 			return ret;
-		}
+		},
+		// Support: IE 11
+		// IE 11 has a bug with offsetWidth / offsetHeight / getClientBoundingRect()
+		// in fullscreen mode inside an iframe the values are 100x smaller than they should be
+		// See https://connect.microsoft.com/IE/feedback/details/838286/ie-11-incorrectly-reports-dom-element-sizes-in-fullscreen-mode-when-fullscreened-element-is-within-an-iframe
```

Comment:
Maybe shorten the URL to just https://connect.microsoft.com/IE/feedback/details/838286 because it's taking a lot of editor columns. :)